xcc-a won't compile

**Description of the bug**
xcc-a builds wasm file but fails when calling `oasis test`. This is after changing the correct path to the xcc-b wasm file (from `debug` to `release`)  in xcc-a's Cargo file.

**Steps to reproduce**
1. `oasis build` on xcc-b
2. `oasis build` on xcc-a
3.  `oasis test` on xcc-a

```
 Compiling xcc-a v0.1.0 (/Users/Kim/oasis-rs/tests/xcc-a)
   Compiling xcc_b v0.1.1 (/Users/Kim/oasis-rs/tests/target/release/build/oasis_imports/xcc_b.rs)
error: extern location for xcc_b does not exist: /Users/Kim/oasis-rs/tests/target/release/deps/libxcc_b-c38e3825f20d95cf.rlib

error[E0463]: can't find crate for `xcc_b`
  --> xcc-a/src/main.rs:11:73
   |
11 |     pub fn call_b(&self, _ctx: &Context, b_addr: Address) -> Result<Vec<xcc_b::Number>, ()> {
   |                                                                         ^^^^^ can't find crate

error: aborting due to 2 previous errors
```

**Expected behavior**
`oasis test` compiles `xcc-a`

**Environment**
 - OS: macOS Mojave

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -85,7 +85,7 @@ jobs:
       - run:
           name: cargo build
           command: |
-            RUSTC_WRAPPER=oasis-build cargo build --target=wasm32-wasi \
+            RUSTC_WRAPPER=oasis-build cargo build --release --target=wasm32-wasi \
               -p ballot \
               -p erc20 \
               -p hello-world \
@@ -102,7 +102,7 @@ jobs:
       - run:
           name: Test IDL generation
           working_directory: tests/idl-gen
-          command: cargo build --target wasm32-wasi --bins && cargo test --lib
+          command: cargo build --release --target wasm32-wasi --bins && cargo test --lib
 
   test_imports:
     executor: rust
@@ -114,11 +114,11 @@ jobs:
       - run:
           name: Build dependency
           working_directory: tests/xcc-b
-          command: cargo build --target wasm32-wasi --bin xcc-b
+          command: cargo build --release --target wasm32-wasi --bin xcc-b
       - run:
           name: Build toplevel
           working_directory: tests/xcc-a
-          command: cargo build --target wasm32-wasi --bin xcc-a && cargo test --lib
+          command: cargo build --release --target wasm32-wasi --bin xcc-a && cargo test --lib
 
 
 workflows:

--- a/tests/idl-gen/src/lib.rs
+++ b/tests/idl-gen/src/lib.rs
@@ -1,6 +1,6 @@
 pub fn test_oasis_interface(bin_name: &str, service_name: &str) {
     let mf_dir = std::path::PathBuf::from(std::env::var("CARGO_MANIFEST_DIR").unwrap());
-    let wasm_path = mf_dir.join(format!("../target/wasm32-wasi/debug/{}.wasm", bin_name));
+    let wasm_path = mf_dir.join(format!("../target/wasm32-wasi/release/{}.wasm", bin_name));
 
     let iface_bytes = walrus::Module::from_file(wasm_path)
         .expect("No wasm")

--- a/tests/xcc-a/Cargo.toml
+++ b/tests/xcc-a/Cargo.toml
@@ -13,4 +13,4 @@ serde = { version = "1.0", features = ["derive"] }
 idl-gen = { path = "../idl-gen" }
 
 [package.metadata.oasis-dependencies]
-xcc-b = "file:///../target/wasm32-wasi/debug/xcc-b.wasm?version=0.1"
+xcc-b = "file:///../target/wasm32-wasi/release/xcc-b.wasm?version=0.1"

